tests/unit_tests/udfs/test_sequence_classification_single_text.py

```diff
@@ -34,8 +34,7 @@ def from_pretrained(cls, model_name, cache_dir):
 
         @property
         def logits(self) -> torch.FloatTensor:
-            return torch.FloatTensor([
-                [0.1, 0.1, 0.1, 0.1], [0.1, 0.2, 0.3, 0.4]])
+            return torch.FloatTensor([[0.1, 0.2, 0.3, 0.4]])
```

Comment:
Now we only have one row per model. We need at best tests for many combinations, and these need to be different tests, probably the best strategy to implement that is via parameterized tests. One parameter, is likely the udf wrapper, because you need different mocks for the model

single model, single batch, batch complete
single model, single batch, batch incomplete
single model, multiple batch, last batch complete
single model, multiple batch, last batch incomplete
multiple model, single batch, last batch complete
multiple model, single batch, last batch incomplete
multiple model, multiple batch, last batch complete
multiple model, multiple batch, last batch incomplete
multiple model, multiple batch, multiple models per batch
multiple model, multiple batch, one model in multiple batches